app/src/App.js

```diff
@@ -1,144 +1,145 @@
 import React from 'react'
-import { useAragonApi, useConnectedAccount } from '@aragon/api-react'
+import { useAragonApi } from '@aragon/api-react'
 import {
-  Main,
   Button,
   IconPlus,
   Header,
   Split,
   GU,
   textStyle,
+  useLayout,
   useTheme,
   Tag,
   SyncIndicator,
 } from '@aragon/ui'
-import styled from 'styled-components'
-import { useAppLogic } from './app-logic'
+import { AppLogicProvider, useAppLogic } from './app-logic'
 import NoWrappedTokens from './screens/NoWrappedTokens'
 import Holders from './screens/Holders'
 import Panel from './components/ActionsPanel'
 import InfoBox from './components/InfoBox'
 import { IdentityProvider } from './components/IdentityManager/IdentityManager'
 
 function App() {
-  const { api, appState } = useAragonApi()
-  const {
-    wrappedTokenSymbol,
-    erc20TokenSymbol,
-    holders,
-    isSyncing,
-    orgTokenAddress,
-    wrappedTokenAddress,
-  } = appState
+  const { appState } = useAragonApi()
+  const { holders, isSyncing, outsideToken, wrappedToken } = appState
```

Comment:
I've changed the schema that the background script provides, so the token information is better enclapsulated:

```js
{
  outsideToken: {
     decimals,
     name,
     symbol,
     totalSupply
  },
  wrappedToken: {
    ...
  },
```